### PR TITLE
Integrate Trivy vulnerability scanner

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
This PR integrates the [Trivy](https://github.com/aquasecurity/trivy) vulnerability scanner for repository file system and it will check and generate the report daily or on demand and before the release.

Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/1849

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: .github/workflows/trivy-scan.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
```
Comment (niveathika) [inline .github/workflows/trivy-scan.yml:20]: We don't need this step now.

File: .github/workflows/central-publish.yml [hunk 2]
```diff
@@ -16,6 +16,18 @@ jobs:
           java-version: 11
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
```
Comment (niveathika) [inline .github/workflows/central-publish.yml:18]: Same

File: .github/workflows/publish-release.yml [hunk 3]
```diff
@@ -17,6 +17,20 @@ jobs:
         with:
           distribution: 'adopt'
           java-version: 11
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+        run: ./gradlew build -x check
```
Comment (ldclakmal) [inline .github/workflows/publish-release.yml:26]:
```suggestion
        run: ./gradlew build -x check -x test
```

File: .github/workflows/central-publish.yml [hunk 4]
```diff
@@ -16,6 +16,18 @@ jobs:
...
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+        run: ./gradlew build -x check
```
Comment (ldclakmal) [inline .github/workflows/central-publish.yml:23]:
```suggestion
        run: ./gradlew build -x check -x test
```

File: .github/workflows/trivy-scan.yml (hunk 1 above)
Comment (ldclakmal) [reply .github/workflows/trivy-scan.yml]: Fixed with d942839

File: .github/workflows/central-publish.yml (hunk 2 above)
Comment (ldclakmal) [reply .github/workflows/central-publish.yml]: Fixed with d942839

**Review by ThisaruGuruge — APPROVED** [review]
LGTM

**Pull request merged**